pytest_fixtures/satellite_auth.py

```diff
@@ -20,6 +20,7 @@
 def ad_data():
     return {
         'ldap_user_name': settings.ldap.username,
+        'ldap_user_cn': settings.ldap.username,
```

Comment:
Why we need to duplicate same value ? In case if you still want to use it make the name as  `ldap_account_name`

Reply:
For ad, username and account name are the same but for openldap and ipa it's not. So while making the tests parameterize, different values were getting passed than required. So in order to make it same I duplicated the value.